<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuInicial</class>
 <widget class="QMainWindow" name="MenuInicial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 0, 153);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>500</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="Solitario">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string>UN SOLO JUGADOR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Multijugador">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string>DOS JUGADORES</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CargarPartida">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string>CARGAR PARTIDA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Instrucciones">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Fixedsys&quot;;</string>
       </property>
       <property name="text">
        <string>INSTRUCCIONES</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>212</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="nombre">
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;Fixedsys&quot;;
background-color: rgba(103, 52, 213, 100);</string>
       </property>
       <property name="text">
        <string>NOMBRE</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="contrasena">
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;Fixedsys&quot;;
background-color: rgba(103, 52, 213, 100);</string>
       </property>
       <property name="text">
        <string>CONTRASEÃ‘A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Continuar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Fixedsys&quot;;</string>
    </property>
    <property name="text">
     <string>CONTINUAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atras">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Fixedsys&quot;;</string>
    </property>
    <property name="text">
     <string>VOLVER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(100, 154, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 700 48pt &quot;Sitka Text&quot;;</string>
    </property>
    <property name="text">
     <string>FINAL DEFENDER</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>800</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>

